protected/humhub/modules/user/tests/codeception/unit/ApproveUserFormTest.php

```diff
@@ -216,6 +291,18 @@ public function testNonGroupManagerCannotDeclineUnapprovedUser()
         $this->assertUnapproved();
     }
 
+    private function assertSendMessage(User $user = null)
+    {
+        if (!$user) {
+            $user = $this->unapprovedUser;
+        }
+
+        $this->assertMailSent(1);
+        $this->assertEqualsLastEmailSubject('About the account request for \'UnApproved User\'.');
+        $this->assertEqualsLastEmailTo($user->email);
+        $this->assertNull(User::findOne(['id' => $this->unapprovedUser->id]));
```

Comment:
Why the unapproved User should be NULL after sending a message?
I don't find where it is deleted, so maybe here we need `$this->assertNotNull(....)` instead.

Reply:
Thanks very much @yurabakhtin ! Yes, I've removed the line, because we don't even need to check if the user exists or not, we only want to test if the message is sent correctly.